source/features/profile-hotkey.tsx

```diff
@@ -12,8 +12,9 @@ function init(): void {
 
 void features.add(__filebasename, {
 	shortcuts: {
-		'g m': 'Go to Profile',
```

Comment:
Let's just send this single line in a new PR if other copy-related changes are included